storage-proofs/Cargo.toml

```diff
@@ -12,7 +12,7 @@ bitvec = "0.5"
 sapling-crypto = { git = "https://github.com/zcash-hackworks/sapling-crypto", branch = "master" }
 rand = "0.4"
 libc = "0.2"
-merkle_light = { git = "https://github.com/filecoin-project/merkle_light", rev = "63a609decf3cc552b0aa79477b828f4a493f725d" }
+merkle_light = { git = "https://github.com/filecoin-project/merkle_light", branch = "mmap-experiments" }
```

Comment:
BLOCKING: lets merge this before merging this.

Reply:
Not sure which "this" is *this*.

Reply:
I suspect @dignifiedquire means merge the `mmap-experiments` branch in `merkle_light` and depend on the resulting `master` commit here.

Reply:
indeed

Reply:
ok, as stated in the PR [description](https://github.com/filecoin-project/rust-fil-proofs/pull/529#issue-259195810) and the ZenHub pipeline that was already the plan, so glad we're on the same page